### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by startupandrew**
@sararob -- please review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
+Since Firebase is a hosted service, your app can use the Firebase JS library to talk to the Firebase servers directly. In combination with our [flexible security rules](https://www.firebase.com/docs/security/quickstart.html) that control access to data, this means that in many cases you don't need any backend code at all to write a production ready app.
 
-### Realtime by default
+### Authentication
 
-Firebase is a realtime data store, which means all data synchronized via the service changes in realtime. When a change occurs on one of your clients, the change propogates to all other clients using your app nearly instantly. This makes Firebase ideal for collaborative web apps where data changes frequently.
+Firebase [Authentication](https://www.firebase.com/docs/web/guide/user-auth.html) makes it easy to add authentication to your app via Facebook, Github, Persona, Twitter, or email / password login. If you already have your own user accounts, Firebase can integrate with your servers as well.
```
Comment (sararob) [inline README.md]: Suggestion: "your own user accounts" -> "your own user authentication system"

File: README.md [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
+> [Firebase](https://www.firebase.com) is a powerful platform for your mobile and web applications that lets you build apps fast without managing servers.
 
 ## Introduction
-This starter pack is for anyone building a web app with Angular and Firebase. Firebase's realtime data store works well with [Angular](http://angularjs.org)'s two-way data binding and content rendering. This repository includes example applications that use some of the common patterns and best practices for integrating the two technologies.
+This starter pack is for anyone building a web app with Angular and Firebase. Firebase's realtime database works well with [Angular](http://angularjs.org)'s two-way data binding and content rendering. This repository includes example applications that use some of the common patterns and best practices for integrating the two technologies.
 
-We recommend taking the tutorial on both [Firebase](https://www.firebase.com/tutorial/) and [AngularJS](http://docs.angularjs.org/tutorial) before proceeding.
+We recommend taking the tutorial on both the [Firebase](https://www.firebase.com/tutorial/) database and [AngularJS](http://docs.angularjs.org/tutorial) before proceeding.
```
Comment (sararob) [inline README.md]: suggestion: "the Firebase database" -> "the Firebase JavaScript client" I'd also make the whole text link to the tutorial

**Comment by sararob** [comment]
@startupandrew two minor comments, otherwise looks good

**Comment by startupandrew** [comment]
Tweaks made. Thanks. PTAL.

**Comment by sararob** [comment]
@startupandrew I updated the link text on line 8, LGTM :+1:

**Pull request merged**